README.md

```diff
@@ -7,9 +7,39 @@ In particular:
 * A regression in Chrome 55 where images get no rectangle when they are wrapped in a node and you select across them.
 * A bug in IE (<=10) which results in scaled rectangles when using the browser's zoom feature.
 
+Install
+=======
+
+```bash
+$ npm install rangefix
+```
+
 Usage
 =====
-Include the rangefix.js library in your project.
+
+**CommonJS**
+
+```javascript
+const RangeFix = require('rangefix');
```

Comment:
code should probably be consistent with our style guide, so no ES6 const, and more spaces:

var RangeFix = require( 'rangefix' );